controllers/apps/transformer_component_account_provision.go

```diff
@@ -222,6 +222,10 @@ func (t *componentAccountProvisionTransformer) provisionAccount(transCtx *compon
 		return nil
 	}
 
+	userInfo, err := lorryCli.DescribeUser(transCtx, string(username))
+	if err == nil && len(userInfo) != 0 {
+		return nil
+	}
```

Comment:
I remember there is a check inside lorry CreateUser, https://github.com/apecloud/kubeblocks/pull/7584/files
When user exists, CreateUser returns success.
So why this check is necessary?

Reply:
Yes, previously we added idempotent support in lorry, but during the upgrade, the pod didn't restart and was still using the old version, so we also need to make adjustments in the Controller.